tests/distributions/test_transform.py

```diff
@@ -631,6 +631,12 @@ def test_univariate_transform_multivariate_dist_raises():
             m.logp(jacobian=jacobian)
 
 
+def test_log_jac_det_ndim_geq_logp_ndim():
```

Comment:
@ricardoV94 I must confess I am a bit lost. I pushed this dummy test (does nothing yet). What do you mean the "default transform" ? (i guess bounds between -1 and 1?). Also, how do I pass the identity (or None) to test the non-transformed?

Reply:
I also see the test `test_univariate_transform_multivariate_dist_raises` is failing because it is not raising enaything because of the change. Should I delete it or how should I modify it? Thanks 🙏

Reply:
You can change the old test and confirm the logp is the expected one, and has the right shape (before it would wrongly be a vector of length 3 instead of a scalar

Reply:
> Also, how do I pass the identity (or None) to test the non-transformed?

You can set `transform=None` when defining the LKJ to disable transform.

To test the default transform just do nothing and ensure `m.rvs_to_transforms[rv]` is not None.

I would call this test something like LKJCorr default transform